msg/QRCodeInfo.msg

```diff
@@ -0,0 +1,5 @@
+builtin_interfaces/Time time_stamp  # timestamp when message was sent
+string sender_id                    # node name of the sending node
+string qr_code_content              # decoded content of the detected QR-Code
+float32 qrcode_position_x           # x postion of the center of the QR-Code relative to the middle of the image from -100% to +100%
```

Comment:
Fände noch relevant zu sagen in welche Richtung sich die Achsen ausbreiten.
Die Y Achse geht in der Mathematik ja gerne nach "oben" und in der Informatik nach "unten".

Reply:
Ist "normal" also mathematisch und da das eh nur ich benutzen werde, reicht das glaube ich hier :D

Reply:
Alles klar.